lib/classes/Variables.js

```diff
@@ -822,6 +827,24 @@ class Variables {
       });
   }
 
+  getValueStrToBool(variableString) {
+    return BbPromise.try(() => {
+      if (_.isString(variableString)) {
+        const groups = variableString.match(this.strToBoolRefSyntax);
+        const param = groups[1].trim();
+          if (param === 'false') {
+            return false;
+          } else if (!param || param === '') {
```

Comment:
I think this is dead condition, as by regex param can't turn `''`, am I missing something?

Reply:
Good point.